### PR TITLE
Typography

### PR DESCRIPTION
**Pull request by carl-robin**
Updated base size from 14px to 16px, "1rem/em = 16px".
We're now using Proxima Nova instead of Lato as our standard font.

### CODE REVIEW

File: app/assets/stylesheets/semantic-ui/elements/_button.scss [hunk 1]
```diff
@@ -55,23 +54,17 @@
 ---------------*/
 
 .ui.button:hover {
-  background-color: #CACBCD;
-  background-image: none;
-  box-shadow: 0px 0px 0px 1px transparent inset, 0px 0em 0px 0px rgba(34, 36, 38, 0.15) inset;
-  color: rgba(0, 0, 0, 0.8);
+  background: #7b8188;
```
Comment (borglin) [inline app/assets/stylesheets/semantic-ui/elements/_button.scss]: Use variables for colors and give them meaningful names, not chalice :P

File: app/assets/stylesheets/semantic-ui/elements/_button.scss [hunk 2]
```diff
@@ -95,19 +87,15 @@
 ---------------*/
 
 .ui.active.button {
-  background-color: #C0C1C2;
-  background-image: none;
-  box-shadow: 0px 0px 0px 1px transparent inset;
-  color: rgba(0, 0, 0, 0.95);
+  background-color: '';
```
Comment (borglin) [inline app/assets/stylesheets/semantic-ui/elements/_button.scss:92]: `''` is not a valid color. Did you mean `transparent`?

File: app/assets/stylesheets/semantic-ui/elements/_button.scss [hunk 3]
```diff
@@ -789,41 +782,40 @@
 .ui.basic.inverted.buttons .button,
 .ui.basic.inverted.button {
   background-color: transparent !important;
-  color: #F9FAFB !important;
-  box-shadow: 0px 0px 0px 2px rgba(255, 255, 255, 0.5) inset !important;
+  color: rgba(0,0,0,.7) !important;
```
Comment (borglin) [inline app/assets/stylesheets/semantic-ui/elements/_button.scss]: Again use varaiables for colors, it makes it so much easier to understand the intent.

File: app/assets/stylesheets/semantic-ui/elements/_button.scss [hunk 4]
```diff
@@ -3299,37 +3287,36 @@
...
 .ui.negative.button {
-  background-color: #DB2828;
+  background: #FF4B4B;
   color: #FFFFFF;
   text-shadow: none;
-  background-image: none;
 }
 .ui.negative.button {
-  box-shadow: 0px 0em 0px 0px rgba(34, 36, 38, 0.15) inset;
+  box-shadow: 0 3px 6px rgba(0,0,0,.15);
```
Comment (borglin) [inline app/assets/stylesheets/semantic-ui/elements/_button.scss]: Use a variable

File: app/assets/stylesheets/semantic-ui/elements/_button.scss [hunk 5]
```diff
@@ -196,9 +184,9 @@
 .ui.disabled.button:hover,
 .ui.disabled.active.button {
   cursor: default;
-  opacity: 0.45 !important;
+  opacity: 0.6 !important;
   background-image: none !important;
-  box-shadow: none !important;
+  box-shadow: 0 3px 6px rgba(0, 0, 0, 0.15);
```
Comment (borglin) [inline app/assets/stylesheets/semantic-ui/elements/_button.scss]: This is a box-shadow definition being used for multiple button styles, replace it with a variable

File: app/assets/stylesheets/semantic-ui/elements/_button.scss [hunk 6]
```diff
@@ -934,13 +926,13 @@
 .ui.toggle.buttons .active.button,
 .ui.buttons .button.toggle.active,
 .ui.button.toggle.active {
-  background-color: #21BA45 !important;
-  box-shadow: none !important;
+  background: #586A81;
   text-shadow: none;
+  box-shadow: 0 3px 6px rgba(0,0,0,.15);
```
Comment (borglin) [inline app/assets/stylesheets/semantic-ui/elements/_button.scss]: Use a variable

File: app/assets/stylesheets/semantic-ui/elements/_segment.scss [hunk 7]
```diff
@@ -660,7 +660,7 @@
 }
 .ui.inverted.segment,
 .ui.primary.inverted.segment {
-  background: #1B1C1D;
+  background: #FFFFFF;
```
Comment (borglin) [inline app/assets/stylesheets/semantic-ui/elements/_segment.scss]: Why use a hex color here and `white` on other places? Please be consistent

File: app/assets/stylesheets/semantic-ui/globals/_variables.scss [hunk 8]
```diff
@@ -1,4 +1,4 @@
 $import-google-fonts: true !default;
-$font-url: 'https://fonts.googleapis.com/css?family=Lato:400,700,400italic,700italic&subset=latin,latin-ext' !default;
-$font-name: 'Lato' !default;
-$font-family: $font-name, 'Helvetica Neue', Arial, Helvetica, sans-serif !default;
+$font-url: 'proxima-nova' !default;
```
Comment (borglin) [inline app/assets/stylesheets/semantic-ui/globals/_variables.scss:2]: This won't work unless you make sure that the font is loaded elsewhere. I don't think everyone watching the docs will have the font installed locally.

**Review by borglin — CHANGES_REQUESTED** [review]
I think we should use variables for most things to help everyone developing achieve a consistent style. Colors are obvious, but also things like margins and paddings could be standardized using variables, in the long run.

File: app/assets/stylesheets/semantic-ui/globals/_variables.scss (hunk 8 above)
Comment (carl-robin) [reply app/assets/stylesheets/semantic-ui/globals/_variables.scss:2]: @borglin The font is loaded via a javascript from Typekit, you don't need it locally.

File: app/assets/stylesheets/semantic-ui/elements/_button.scss (hunk 2 above)
Comment (carl-robin) [reply app/assets/stylesheets/semantic-ui/elements/_button.scss:92]: Semantic already uses '' in some places, thats why I continued.

**Pull request closed without merging**